libraries/core/src/test/java/com/paritytrading/philadelphia/FIXInitiatorSocketTest.java

```diff
@@ -0,0 +1,26 @@
+package com.paritytrading.philadelphia;
+
+import java.io.IOException;
+import java.nio.channels.ServerSocketChannel;
+import java.nio.channels.SocketChannel;
+import org.jetbrains.annotations.NotNull;
+
+public class FIXInitiatorSocketTest extends FIXInitiatorTest<SocketChannel> {
```

Comment:
What do you think if we called this `FIXInitiatorTestWithSocketChannel`?

Let's make also this class package-private.

Reply:
See my reply from the Pipe test.